src/main/kotlin/no/nav/familie/klage/ekstern/EksternBehandlingController.kt

```diff
@@ -53,14 +53,46 @@ class EksternBehandlingController(
         val antallTreff = behandlinger.entries.associate { it.key to it.value.size }
         logger.info("Henter klagebehandlingsresultat for eksternFagsakIder=$eksternFagsakIder antallTreff=$antallTreff")
         validerTilgang(behandlinger)
+        return Ressurs.success(behandlinger)
+    }
+
+    @GetMapping("/baks/{fagsystem}")
+    fun hentBehandlingerBAKS(
+        @PathVariable fagsystem: Fagsystem,
+        @RequestParam("eksternFagsakId") eksternFagsakId: String,
+    ): Ressurs<List<KlagebehandlingDto>> {
+        feilHvis(eksternFagsakId.isBlank()) {
+            "Mangler eksternFagsakId i query param"
+        }
+
+        feilHvis(fagsystem !in listOf(Fagsystem.BA, Fagsystem.KS)) {
+            "Ugyldig fagsystem: $fagsystem. Endepunkt støtter kun BA og KS."
+        }
+
+        val behandlinger = hentBehandlingerBAKS(eksternFagsakId, fagsystem)
 
         return Ressurs.success(behandlinger)
     }
```

Comment:
Legger til eget endepunkt for henting av klagebehandlinger for BAKS. For oss vil `eksternFagsakId` alltid kun være 1 fagsak.